localMatchApp/sources/Login.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Login.h"
+
+using namespace std;
+
+Login::Login ( ) {
+  // Connect to database.
+}
+
+Login::~Login ( ) {
+  // Disconnect to database.
+}
+
+Login::connectionMessage Login::signUp ( const string i_user, const string i_password,
+                                         const string i_repeatedPassword ) {
+  // Send information to database.
+  string userInDatabase = "";
+
+  if ( hasCorrectFormat ( i_password ) && hasCorrectFormat ( i_repeatedPassword ) ) {
```

Comment:
To avoid nesting IF statements, somethings it is easier to check the failure condition like:
```c
if ( !hasCorrectFormat ( i_password ) || !hasCorrectFormat ( i_repeatedPassword ) ) {
  return connectionMessage::wrongPassword;
}

if ( i_password != i_repeatedPassword ) {
  return connectionMessage::notEqualPasswords;
}

if ( i_user.compare ( userInDatabase ) != 0 ) {
  return connectionMessage::ocupatedUser;
}

// Set user and password in table
return connectionMessage::goodEntry;
```

Reply:
Done.